Sources/Danger/GitLabDSL.swift

```diff
@@ -53,7 +92,9 @@ extension GitLab {
             public let dueDate: Date?
             public let id: Int // swiftlint:disable:this identifier_name
```

Comment:
<!--
  0 failure: 
  1 warning:  SwiftLint rule 'i...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitLabDSL.swift;
  Line: 93;
-->

- :warning: SwiftLint rule 'identifier_name' did not trigger a violation in the disabled region. Please remove the disable command. (`superfluous_disable_command`)